### PR TITLE
kops repos to require self-approval, approve button = lgtm

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ approve:
   - kubernetes-client
   - kubernetes-csi
   - kubernetes-sigs
-  require_self_approval: false
+  require_self_approval: true
```
Comment (spiffxp) [inline config/prow/plugins.yaml:75]: you probably don't mean to be changing this for `kubernetes/kubernetes`, every repo in `kubernetes-sigs`, etc.

move `kubernetes/kops` to the entry above that contains `kubernetes/website`

**Review by spiffxp — CHANGES_REQUESTED** [review]
/hold
don't want this to accidentally merge

File: config/prow/plugins.yaml (hunk 1 above)
Comment (olemarkus) [reply config/prow/plugins.yaml]: oups. I most certainly did not.

Thanks. Moved.

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold cancel

**Pull request merged**